### DISCUSSION TITLE
ternary comparison operator

### DISCUSSION THREAD

**Discussion by theking2** [dotnet/csharplang#7050] [Language Ideas] [answered]
add suport for ternary comparison operators:
produktion rule;
```
expression
   : simpleExpression relationaloperator simpleExpression relationaloperator simpleExpression
   | simpleExpression (relationaloperator expression)?
   ;
```

This would allow to write something like
```
x = 10;
t = 10 <= x < 100;
```

Where variable x is compaired to 10 including and 100 excluding.

Benefit: x is evaluated only once, where instead the simalar:
```
t = 10 <= x && x < 100; 
```
would evaluate x twice.

**Comment by HaloFour** [comment, marked as answer]
See: https://github.com/dotnet/csharplang/issues/4108
